### PR TITLE
Feat/civ 6176

### PR DESCRIPTION
**Pull request by lfazelyniaHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-6176

### Change description ###
This change will make sure that the existing page (first one in I2P journey) also covers the "Full Admit and already paid" scenario by amending FieldShowConditions and adding the necessary new labels

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/CaseField/CaseFieldLRspec-CUI-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
+[
```
Comment (lfazelyniaHMCTS) [inline ccd-definition/CaseField/CaseFieldLRspec-CUI-nonprod.json:1]: **respondent1FullAdmitPaidLabelSpec** and **applicant1FullAdmitConfirmAmountPaidSpec** are CIV-6176 changes

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldLRspec-CUI-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,273 @@
+[
```
Comment (lfazelyniaHMCTS) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldLRspec-CUI-nonprod.json:1]: CIV-6176 changes:
- respondent1FullAdmitPaidLabelSpec
- applicant1FullAdmitConfirmAmountPaidSpec

File: ccd-definition/AuthorisationCaseField/staffLRspec-CUI-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
+[
```
Comment (lfazelyniaHMCTS) [inline ccd-definition/AuthorisationCaseField/staffLRspec-CUI-nonprod.json:1]: CIV-6176 changes:
- respondent1FullAdmitPaidLabelSpec
- applicant1FullAdmitConfirmAmountPaidSpec

File: ccd-definition/AuthorisationCaseField/systemUpdateLRspec-CUI-nonprod.json [hunk 4]
```diff
@@ -0,0 +1,80 @@
+[
```
Comment (lfazelyniaHMCTS) [inline ccd-definition/AuthorisationCaseField/systemUpdateLRspec-CUI-nonprod.json:1]: CIV-6176 changes:
- respondent1FullAdmitPaidLabelSpec
- applicant1FullAdmitConfirmAmountPaidSpec

File: ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json [hunk 5]
```diff
@@ -0,0 +1,367 @@
+[
```
Comment (lfazelyniaHMCTS) [inline ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json:1]: CIV-6176 changes:
- respondent1FullAdmitPaidLabelSpec
- applicant1FullAdmitConfirmAmountPaidSpec
- Added fieldshowcondition **showResponseOneVOneFlag!=\"ONE_V_ONE_FULL_ADMIT_HAS_PAID\"** on a few elements
- Added fieldshowcondition **showResponseOneVOneFlag=\"ONE_V_ONE_FULL_ADMIT_HAS_PAID\"** on a few elements

File: e2e/fixtures/events/claimantResponseSpecCui.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+        };
+        break;
+      case 'PART_ADMISSION':
+        responseData.userInput = {
+          ...responseData.userInput,
+          RespondentResponse: {
+            applicant1ProceedWithClaim: 'Yes',
+            applicant1AcceptAdmitAmountPaidSpec: 'Yes',
+            applicant1AcceptPartAdmitPaymentPlanSpec: 'No',
+            applicant1FullAdmitConfirmAmountPaidSpec: 'Yes'
```
Comment (lfazelyniaHMCTS) [inline e2e/fixtures/events/claimantResponseSpecCui.js:84]: CIV-6176 changes:
- applicant1FullAdmitConfirmAmountPaidSpec

**Review by laperla11 — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Pull request closed without merging**